.dockerignore

```diff
@@ -5,7 +5,7 @@
 !public/
 !sample-data/
 !src/
-
+node_modules/.prisma/
```

Comment:
```suggestion
!node_modules/.prisma/
```

Reply:
Actually, this doesn't make sense either way... node_modules/.prisma isn't committed to git so there's no effect from whitelisting `!...` or blacklisting it.

Reply:
I'll remove it and see if everything still works.  I added this at the point when the docker build was broken and we didn't know why.  It should all just work without the explicit call to `prisma generate`, but that's not the case.